api.planx.uk/admin/feedback/downloadFeedbackCSV.ts

```diff
@@ -61,8 +61,6 @@ type ParsedFeedback = Feedback & {
  *    description: Downloads the FeedbackFish CSV
  *    tags:
  *      - admin
- *    security:
- *      - bearerAuth: []
```

Comment:
Not sure if I completely understand why the `security` schema key is being completely removed from admin endpoints, rather than replaced with `cookieAuth: []` ?

When an endpoint has this `security` key, swagger renders the :lock: icon which is a nice visual indicator of our authenticated versus public endpoints - I'm wondering if there's a way to keep that visual queue while using this simplified auth strategy? Not a dealbreaker, but just wondering!

Reply:
So this is a good catch and an interesting one.

I removed this on all individual endpoints, but added to the root of the docs to apply it across all of them (`server.ts`, line 316)

This does apply the lock icon to most endpoints, but not all for some reason - 

<img width="1434" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/1e12d4f3-8d93-4538-bf23-1347caf0817f">

The best way to handle this is to just tag each endpoint manually I think - I'll do that now